sw/device/tests/ottf_example_bare_metal_test.c

```diff
@@ -0,0 +1,17 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <sw/device/lib/testing/test_framework/ottf.h>
```

Comment:
Uhhh why is this in `<>`? I'm not sure I had noticed this before but _only_ the libc headers should be in `-isystem`.

Reply:
Oops! Not sure how that ended up like that.

Fixed.